server/client.go

```diff
@@ -596,6 +608,13 @@ func (c *client) setPermissions(perms *Permissions) {
 		}
 	}
 
+	// Check if we are allowed to send responses.
+	if perms.Response != nil {
+		c.perms.resp = perms.Response
```

Comment:
Is that safe? Should we make a copy/clone of perms.Response? I am thinking config reload, if permissions were to change. Maybe it's not relevant, but for other perms with insert in local sublist, here we store the reference..

Reply:
We could make a copy, we do not have sublists it just 2 fields, and int and a time.Duration.

Reply:
ok fixed. Will squash down all the commits when we sign off.